setup.py

```diff
@@ -198,5 +200,13 @@ def read(*names, **kwargs):
             'scancode = scancode.cli:scancode',
             'extractcode = scancode.extract_cli:extractcode',
         ],
+        'scancode_plugins': [
+            'formats = plugincode.formats',
```

Comment:
It's the `scan_proper` plugin which calls all other `scan_output` formats are returns the `--format` option